src/bundle/Controller/LocationController.php

```diff
@@ -68,4 +70,37 @@ public function actionsAction(
             )
         );
     }
+
+    public function updateSubItemOrderAction(
```

Comment:
I know the story title is misleading but you actually set the Location default sort order and sort field which is then used by _default_ by subitems so I think this should be called `updateLocationDefaultSortOrderField` or something like that